swan-lake/learn-the-language/distinctive-language-features/advanced-general-purpose-language-features.md

```diff
@@ -666,6 +666,89 @@ int cp = ch.toCodePointInt();
 
 In the above code example, **``ch``** is of the ``string:Char`` type. You can also extract the code point of the character by calling the **``toCodePointInt()``** function. This ``string:Char`` subtype is analogous to the XML subtypes like XML elements or text.
 
+## Built-in RegExp type
```

Comment:
Shall we have the heading as just `Regular Expressions`

Reply:
@MaryamZi any thoughts on any other alternative section to include this

Reply:
+1 to use "Regular Expressions". I guess we have the freedom of adding it wherever we want here. Even at the very top or maybe before the section on `never`?

Reply:
Added after the `Char` built-in type section